dist/playkit.js

```diff
@@ -0,0 +1,18483 @@
+(function webpackUniversalModuleDefinition(root, factory) {
```

Comment:
dist folder should not be included - revert this file
check your `.gotignore` or your git tool as this was not supposed to be included at all

Reply:
`.gitignore` suppose to be in git, I think there is a mistake in `.gitignore` we should add dist/* instead dist/ for ignoring this folder at all.
Would we change it on build scenario only, right?